muesli/web/templates/tutorial/add.pt

```diff
@@ -8,26 +8,44 @@
 
 <p tal:condition="error_msg" class="error">${error_msg}</p>
 
-
 <form action="/tutorial/add" tal:attributes="action request.route_path('tutorial_add', lecture_id=lecture.id)" method="POST">
-<metal:main metal:use-macro="templates('Fragments/HTML/form.pt').macros['formmacro']" />
+  <metal:main metal:use-macro="templates('Fragments/HTML/form.pt').macros['formmacro']" />
 </form>
 
 <h4>Bereits existierende ${names['tutorials']}</h4>
-<table>
-  <tr><th>Zeit</th><th>Raum</th><th>${names['tutor']}</th><th>${names['comment']}</th><th></th><th></th></tr>
-  <tr tal:repeat="tutorial lecture.tutorials">
-    <td>${tutorial.time}</td>
-    <td>${tutorial.place}</td>
-    <td>
-      <span tal:condition="tutorial.tutor" tal:content="tutorial.tutor.name()" tal:omit-tag="" />
-    </td>
-    <td>${tutorial.comment}</td>
-    <td><a tal:attributes="href request.route_path('tutorial_view', tutorial_ids=tutorial.id)">Details</a></td>
-    <td><a tal:attributes="href request.route_path('tutorial_edit', tutorial_id=tutorial.id)">Bearbeiten</a></td>
-  </tr>
+<table class="table">
+  <thead class="thead-dark">
+    <th scope="col">Zeit</th>
+    <th scope="col">Raum</th>
+    <th scope="col">${names['tutor']}</th>
+    <th scope="col">${names['comment']}</th>
+    <th scope="col"></th>
+    <th scope="col"></th>
+  </thead>
+  <tbody>
+    <tr tal:repeat="tutorial lecture.tutorials">
+      <td>${tutorial.time}</td>
+      <td>${tutorial.place}</td>
+      <td>
+        <span tal:condition="tutorial.tutor" tal:content="tutorial.tutor.name" tal:omit-tag="" />
+        <a role="button" class="btn-sm btn-success"
+           tal:condition="not tutorial.tutor"
+           tal:attributes="href request.route_path('tutorial_take', tutorial_id=tutorial.id)">Übernehmen</a>
```

Comment:
Clicking this should allow to stay on the tutorial add page instead of going to the lecture

Reply:
Yeah I would agree, but thats nothing I changed :grimacing:

Reply:
I opened issue #126 for that.

Can be resolved.